tests/features/timeline.feature

```diff
@@ -11,12 +11,35 @@ Feature: Timeline paragraph.
     And I fill in "Title" with "Timeline paragraph test"
     And I press "Add Timeline"
     Then the following fields should be present "Label, Title, Content, Expand button"
-
-    When I press "Save"
+    When I fill in "Label" with "Label" in the "first" "Items" field element
+    And I press "Save"
+    Then I should see the following error messages:
+      | error messages                                   |
+      | Title field is required if there is Label input. |
+    When I fill in "Label" with "" in the "first" "Items" field element
```

Comment:
```suggestion
    When I fill in "Label" with "" in the first "Items" field element
```
It's the same, but no need to quote single values and it makes it a bit more understandable.